deployment.go

```diff
@@ -44,6 +46,14 @@ type DeploymentStep struct {
 	App    string `json:"app"`
 }
 
+// DeploymentStepV1 is a Marathon v1.X-compatible DeploymentStep
+type DeploymentStepV1 struct {
+	Actions []struct {
+		Type string `json:"type"`
```

Comment:
I believe the string type and JSON tag should be called `Action` and `action`, respectively: This is what [upstream changed it to](https://github.com/mesosphere/marathon/commit/595eba7d86dd59bb85ee30c03ffd3481136c4dcf). See also @kesselborn's [comparison on the upstream issue](https://github.com/mesosphere/marathon/issues/3820#issuecomment-222201282).

Tests would need to be updated as well.